instance_control/readme.md

```diff
@@ -10,15 +19,15 @@ chmod u+x bubuku_cluster.py
 ```
 
 ### Create
-
+The command is used to create a complete new cluster or add new nodes to the existing cluster (--cluster-size defines how many nodes you want to create/add to the cluster for provided config)
```

Comment:
In order to create the first node of the cluster you have to create security group, iam profile, prepare taupage user data. So, for the first node in the cluster it is really creating something else except node itself, but for the next runs it will just create new nodes using existing configs.